sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/java/com/azure/messaging/eventhubs/stress/util/ScenarioOptions.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.stress.util;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+@Configuration
+public class ScenarioOptions {
+    @Autowired
+    private Environment env;
+
+    @Autowired
+    private ApplicationArguments args;
+
+    public String get(String name) {
```

Comment:
One powerful thing in Spring is configuring variables. I saw many instances of `options.get`. Maybe we can use this feature so it can be easily refactored and followed:
https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config

in this case, you would have `ScenarioOptions.getConsumerGroup()` rather than ScenarioOptions.get("CONSUMER_GROUP")`

Reply:
Sure, I changed to use spring inject all environment/arguments variables. 
Before I was struggling whether to involve spring framework as external dependency. But since it was already used for configuring some beans like `RateMeter`, it is good to highly leverage spring to configure for this project.